matrix-check-inputs/README.md

```diff
@@ -0,0 +1,19 @@
+# matrix-check-inputs
+
+This action is designed to be used to check the incoming
+inputs (dynamically generated).
+
+This is useful to use to test the functionality of `matrix-extra-inputs`.
```

Comment:
You should be able to add tests for this by invoking these as part of `.github/workflows` on this repo 🤞 

We do that for a few of the things in `chainguard-dev/actions` now.

Reply:
done!

Reply:
ive removed this action entirely